sources/packages/backend/libs/utilities/src/esdc-utils.ts

```diff
@@ -71,3 +71,21 @@ export function getCountryCode(country: string): string {
   }
   return country;
 }
+
+/**
+ * Returns a formatted postal code `A#A #A#` only for Canada addresses.
+ * @param country country for the address.
+ * @param postalCode postal code to be formatted.
+ * @returns a formatted postal code.
+ */
+export function getFormattedPostalCode(
+  country: string,
+  postalCode: string,
+): string {
+  if (country.toUpperCase() === "CANADA" && postalCode) {
```

Comment:
Can we add the expected length to the validation, as below where `CANADA_POSTAL_CODE_LENGTH` will be defined as 6.
```ts
if (country.toUpperCase() === "CANADA" && postalCode?.length === CANADA_POSTAL_CODE_LENGTH)
```
IMO anything different than a string with 6 characters should not have the space added.